File: app/src/main/java/com/pgssoft/mvvm/services/interfaces/ApiService.java
package com.pgssoft.mvvm.services.interfaces;

import com.pgssoft.mvvm.model.api.ApiTable;
/**
 * Created by bstokrocki on 29.01.2017.
 */
public interface ApiService {
    void loadRates(ApiServiceCallback<ApiTable[]> callback);
}

File: kotlin/app/src/main/java/com/pgssoft/kotlinplayground/service/iface/ApiService.kt
package com.pgssoft.kotlinplayground.service.iface

import com.pgssoft.kotlinplayground.model.api.ApiTable

/**
 * Created by bstokrocki on 29.01.2017.
 */
interface ApiService {
    fun loadRates(callback: ApiServiceCallback<Array<ApiTable>>)
}
